aspnetcore/blazor/components.md

```diff
@@ -1051,6 +1040,40 @@ In the sample app, the `CascadingValuesParametersTheme` component binds the `The
 }
 ```
 
+To cascade multiple values of the same type within the same subtree, provide a unique `Name` string to each `CascadingValue` component and its corresponding `CascadingParameter`. In the following example, two `CascadingValue` components cascade different instances of `MyCascadingParameterType` by name:
+
+```cshtml
+<CascadingValue Value=@ParentCascadeParameter1 Name="CascadeParam1">
+    ...
+</CascadingValue>
+
+<CascadingValue Value=@ParentCascadeParameter2 Name="CascadeParam2">
+    ...
+</CascadingValue>
+
+@code {
+    [Parameter]
```

Comment:
I would remove `[Parameter]` on both. It could be a field, prop or parameter.
It kinda suggests that the CascadingValue should have a `[Parameter]` attribute.

Reply:
The `CascadingValue`'s should be nested, but perhaps you put them side-by-side for readability.

```
<CascadingValue Value=@ParentCascadeParameter1 Name="CascadeParam1">
   <CascadingValue Value=@ParentCascadeParameter2 Name="CascadeParam2">
    ...
    </CascadingValue>
</CascadingValue>
```